project/src/main/java/com/google/sps/utilities/SpannerTasks.java

```diff
@@ -450,4 +451,43 @@ public Void run(TransactionContext transaction) throws Exception {
   private static Mutation.WriteBuilder newInsertBuilderFromOpportunitySignup() {
     return Mutation.newInsertBuilder(OPPORTUNITY_SIGNUP_TABLE);
   }
+
+  /**
+   * Given filters for events, return events whose labels match those filters
+   *
+   * @param email of loggedIn user email
```

Comment:
I think the param email is incorrectly described.

Reply:
done